style.css

```diff
@@ -39,13 +41,85 @@
     margin-bottom: 12px;
   }
 
-  .humburger {
+  .navi-menu {
+    display: flex;
+    justify-content: space-between;
+    list-style-type: none;
+    position: fixed;
+    left: -100%;
+    height: 100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #3c3a39;
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+  }
```

Comment:
- [ ] According to the requirements, The mobile menu should  follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) (e.g., font, colors, images, text, margins), so the logo should disappear when you open the menu, and kindly decrease the space between the menu-items to match your Figma design 👍💯

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/195069864-7cf5dc1b-6290-4bb5-8ec8-03311c62bc97.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/195069806-87b10ca6-f32e-40e9-88e0-49d6d2ac18ba.PNG" width="500" /> |

Reply:
I corrected the spacing